files/image_config/sudoers/sudoers

```diff
@@ -38,7 +38,8 @@ Cmnd_Alias      READ_ONLY_CMDS = /bin/cat /var/log/syslog*, \
                                  /usr/local/bin/psuutil *, \
                                  /usr/local/bin/sonic-installer list, \
                                  /usr/local/bin/sfputil show *, \
-                                 /bin/ip netns identify [0-9]*
+                                 /bin/ip netns identify [0-9]*, \
+                                 /usr/local/bin/ipintutil
```

Comment:
Suggest inserting in alphabetical order. Makes it easier to locate.

Reply:
fixed in latest commit